curvefs/src/client/fuse_client.cpp

```diff
@@ -964,11 +964,13 @@ CURVEFS_ERROR FuseClient::FuseOpGetAttr(fuse_req_t req, fuse_ino_t ino,
                                         struct stat *attr) {
     VLOG(1) << "FuseOpGetAttr ino = " << ino;
     if (FLAGS_enableCto) {
-        CURVEFS_ERROR ret = inodeManager_->RefreshInode(ino);
-        if (ret != CURVEFS_ERROR::OK) {
-            LOG(ERROR) << "inodeManager get inode fail, ret = " << ret
-                       << ", inodeid = " << ino;
-            return ret;
+        if (fi == nullptr || fi->fh != 1) {
```

Comment:
what's the initialize value of fi->fh?

Reply:
0 the default value of uint64_t and have changed to enum value which is more meaningful.